src/main/java/core/basesyntax/strategy/OperationStrategyImpl.java

```diff
@@ -4,13 +4,15 @@
 import java.util.Map;
 
 public class OperationStrategyImpl implements OperationStrategy {
+    private static final int OPERATION_INDEX = 0;
+    private static final int FRUIT_INDEX = 1;
+    private static final int QUANTITY_INDEX = 2;
+
     @Override
-    public void accept(List<String> data, Map<String, OperationHandler> operationHandlerMap) {
-        data.remove(0);
-        for (String value : data) {
-            int indexOfSeparator = value.indexOf(OperationHandler.COMMA_SEPARATOR);
-            operationHandlerMap.get(value.substring(0, indexOfSeparator))
-                    .accept(value.substring(indexOfSeparator + 1));
+    public void accept(List<String[]> data, Map<String, OperationHandler> operationHandlerMap) {
+        for (String[] value : data) {
+            operationHandlerMap.get(value[OPERATION_INDEX])
+                    .accept(value[FRUIT_INDEX], value[QUANTITY_INDEX]);
         }
```

Comment:
yeah, now it looks better. you can change this piece of code on .stream(), but overall is good. approve :)